utilities/rpp-performancetests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -1029,6 +1032,74 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 84:
+        {
+            test_case_name = "spatter";
+
+            RpptRGB spatterColor;
+
+            // Mud Spatter
+            spatterColor.R = 65;
+            spatterColor.G = 50;
+            spatterColor.B = 23;
+
+            // Blood Spatter
+            // spatterColor.R = 98;
+            // spatterColor.G = 3;
+            // spatterColor.B = 3;
+
+            // Ink Spatter
+            // spatterColor.R = 5;
+            // spatterColor.G = 20;
+            // spatterColor.B = 64;
+
+            // Uncomment to run test case with an xywhROI override
```

Comment:
we talked about these comments before that you would need to pass these as parameters to the test. When are you going to make these changes?

Reply:
Since every function in the test suite has a different number and type of parameters taken from the user, this change might not actually be a direct one. Some functions take a single u8/uint/f32 or some functions take whole arrays passed in, and others have tensor based ROIs. Will take a look on this in a separate test suite PR. Should be after the next couple of essential PRs ricap/resize/colortwist/cmn are merged.